playbook/spec/dummy/config/initializers/content_security_policy.rb

```diff
@@ -1,25 +1,17 @@
-# Be sure to restart your server when you modify this file.
+Rails.application.config.content_security_policy do |p|
+  p.default_src :self, :https, :unsafe_inline
+  p.font_src    :self, :https, :data, :unsafe_inline
+  p.img_src     :self, :https, :data, :unsafe_inline
+  p.object_src  :none
+  p.script_src  :self, :https, :unsafe_inline
+  p.style_src   :self, :https, :unsafe_inline
 
-# Define an application-wide content security policy
-# For further information see the following documentation
-# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+  # To allow connections to the webpack-dev-server running in
+  # a separate docker container
+  if Rails.env.development?
+    p.connect_src :self, :https, 'http://localhost:3040', 'ws://localhost:3040', 'http://0.0.0.0:3040', 'ws://0.0.0.0:3040'
```

Comment:
Does this mean we might have HMR now?

Reply:
This is the same thing we already had before, so if it didn't work before, this won't make it better, unfortunately. But it's good to know that we don't currently have it and that we'd like to. I'll try to make it happen when I separate the lib from the website.